### PR TITLE
register a lot of shortcuts

### PR DESCRIPTION
**Pull request by jjavierdguezas**
register a lot of shortcuts
- Zoom In
- Zoom Out
- Zoom Reset
- Edit URL
- Scroll Up
- Scroll Down
- Take Screenshot
- Tilt Devices
- Toggle Inspector
- Go to Homepage
- Back a Page
- Forward a Page
- Delete Storage
- Delete Cookies

solve #90

### CODE REVIEW

File: desktop-app/app/containers/Root.js [hunk 1]
```diff
@@ -57,19 +69,67 @@ const getApp = history => {
...
 export default class Root extends Component<Props> {
   componentDidMount() {
-    // const {store} = this.props;
+    this.registerAllShortcuts();
+  }
+
+  registerAllShortcuts = () => {
+    const {store} = this.props;
+
+    registerShortcut({id: 'ZoomIn', title: 'Zoom In', accelerators: ['mod+plus', 'mod+shift+=']}, () => {
```
Comment (manojVivek) [inline desktop-app/app/containers/Root.js:78]: @jjavierdguezas Just curious, which key does the `mod` here refer too?

Comment (jjavierdguezas) [reply desktop-app/app/containers/Root.js:78]: this is the Mousetrap sintax, it is the equivalent of `CommandOrCtrl` you can get more info [here](https://craig.is/killing/mice)

Comment (manojVivek) [reply desktop-app/app/containers/Root.js:78]: That is great, thanks for the info JJ.

**Comment by manojVivek** [comment]
@jjavierdguezas This is great work here adding shortcuts for all primary use cases. 🚀 
Merging this in.

**Comment by jjavierdguezas** [comment]
> @jjavierdguezas This is great work here adding shortcuts for all primary use cases. 🚀
> Merging this in.

@manojVivek Yeah, this PR can serve as a guide to register future shortcuts. Note that they are automatically displayed in the hotkey dialog. Theoretically we can associate a shortcut to every store action. The next step could be develop a shortcut manager UI, where the user defines his owns shortcuts for every registered action

**Comment by manojVivek** [comment]
@jjavierdguezas Yes, that will be a great win for the users. 💪

**Pull request merged**